internal/services/dashboard/dashboard_grafana_resource.go

```diff
@@ -210,6 +223,7 @@ func (r DashboardGrafanaResource) Create() sdk.ResourceFunc {
 					AutoGeneratedDomainNameLabelScope: &model.AutoGeneratedDomainNameLabelScope,
 					DeterministicOutboundIP:           &deterministicOutboundIP,
 					GrafanaIntegrations:               expandGrafanaIntegrationsModel(model.AzureMonitorWorkspaceIntegrations),
+					GrafanaMajorVersion:               &grafanaMajorVersion,
```

Comment:
Since this is available on the Model defined above, we can use that rather than pulling this from the config directly:

```suggestion
					GrafanaMajorVersion:               pointer.To(model.GrafanaMajorVersion),
```